MSAL/src/MSALResult.m

```diff
@@ -135,10 +135,6 @@ + (MSALResult *)resultWithMSIDTokenResult:(MSIDTokenResult *)tokenResult
     }
     
     NSString *accessToken = [authScheme getClientAccessToken:tokenResult.accessToken popManager:popManager error:error];
```

Comment:
To really distinguish the case no accessToken vs parsing accessToken error here, I think we should change to:
if (tokenResult.accessToken is not empty)
{
    // parsing access token
}  else {
// move on to the next step.
}

Reply:
Addressed, thanks.